index.js

```diff
@@ -1,17 +1,49 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort((a, b) => a - b);
+    this.length = this.length;
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (pos >= this.items.length) {
+      throw new Error("EmptySortedList");
+    }
 
-  max() {}
+    return this.items.indexOf(pos);
```

Comment:
mmmm esto no hace lo que quieres no? el indexOf lo que hace es devolverte la primera posicion que se encuentra con ese elemento a buscar, pero nosotros queremos el elemento que se encuentra en la posicion N
Asi que seria mas asi :D
```suggestion
    return this.items[pos];
```